src/org/traccar/api/resource/ReportResource.java

```diff
@@ -32,12 +32,14 @@ public Response getRouteJson(
 
     @Path("route")
     @GET
-    @Produces("application/ms-excel")
+    @Produces("text/csv")
     public Response getRouteCsv(
             @QueryParam("deviceId") final List<Long> deviceIds, @QueryParam("groupId") final List<Long> groupIds,
             @QueryParam("from") String from, @QueryParam("to") String to) throws SQLException {
         return Response.ok(Route.getCsv(getUserId(), deviceIds, groupIds,
-                JsonConverter.parseDate(from), JsonConverter.parseDate(to))).build();
+                JsonConverter.parseDate(from), JsonConverter.parseDate(to)))
+                .header("Content-Disposition", "attachment; filename=route.csv")
```

Comment:
It would be nice to use some constants here. Header name must be available as constant somewhere already. Value can be a constant here (we can just use same filename like `report.csv`).